client/src/partials/depots_management/depots_management.js

```diff
@@ -0,0 +1,131 @@
+angular.module('bhima.controllers')
+.controller('DepotManagementController', DepotManagementController);
+
+DepotManagementController.$inject = ['$translate', 'DepotService', 'SessionService'];
+
+/**
+ * Depot Management Controller
+ * This controller is about the depot management module in the admin zone
+ * It's responsible for creating, editing and updating a depot
+ */
+function DepotManagementController($translate, DepotService, SessionService) {
+  'use strict';
```

Comment:
In the future, you do not need to write 'use strict' in your controllers -- we automatically wrap the entire application in strict mode during the build process.